### PR TITLE
Initial draft of call for reproducible research champions

### PR DESCRIPTION
**Pull request by martintoreilly**
@KirstieJane First pass in `features/2-champions-call` branch linked to this PR. Feel free to edit / hack / completely rewrite as you like.

A couple of questions that arose while writing this draft.
- Should the deadline be midnight Sunday?
- Can you review my first pass estimates of the time commitment required from champions and the amount of RSE support we're promising (we've about 16 weeks total in our remaining budget)?
- Reproducible Research Champions is quite long winded and I'm not sure about using Champions alone (which I've done a lot). What do you think about using Reproducibility Champions everywhere?

### CODE REVIEW

**Comment by KirstieJane** [comment]
Thanks @martintoreilly! I'm just finishing up this meeting - I need to eat and then I'll take a look before my meeting at 1:30pm ✨

File: call-for-champions.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Call for Turing Reproducible Research Champions
+**Application deadline:** Friday 23rd February 2018
```
Comment (KirstieJane) [inline call-for-champions.md]: Great - happy with that deadline 👍

File: call-for-champions.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## What is a Reproducible Research Champion?
+We're looking for Turing researchers who want to take their research to the
+next level in terms of reproducibility and help us establish a culture of
+reproducible research at the Turing.
+
+Enthusiasm for reproducible research is the most important factor. There are many
+reasons that researchers who believe in reproducible research struggle to make
+their research reproducible in practice, and Champions will receive significant
+support from the Turing Research Engineering team to overcome these barriers to
```
Comment (KirstieJane) [inline call-for-champions.md]: Research Software Engineering team? Did you drop the software on purpose? For length?

I don't really mind TBH, just curious :smile:

File: call-for-champions.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# Call for Turing Reproducible Research Champions
+**Application deadline:** Friday 23rd February 2018
+
+## What is a Reproducible Research Champion?
```
Comment (KirstieJane) [inline call-for-champions.md]: We don't define reproducible research...should go up near the top somewhere.

File: call-for-champions.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+sessions will also help other researchers in making their own research
+reproducible.
+
+## What will be asked of me as a champion?
+**A paper:** You must have a paper you want to make reproducible that
+is either published or submitted for publication (pre-prints count).
+
+**Methods, data and code:** You must be happy for the detailed methods, data and
+code associated with your paper to be made available to other researchers, and
+ideally the public at large. We understand that not all data can be made available
```
Comment (KirstieJane) [inline call-for-champions.md]: replace:

> not all data can be made available in a completely open manner

with

> not all data can be made publicly available

**Review by KirstieJane — APPROVED** [review]
AWESOME @martintoreilly!

Sorry I didn't get to this earlier today.

File: call-for-champions.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+- Attending a subset of the reproducible research drop-in sessions to help other
+researchers in making their own research reproducible. **[6-10 hours total]**
+- The publicity campaign highlighting reproducible research best practice at the
+Turing. This will require writing a popular summary for your paper and
+contributing to publicity material such as blog posts, articles and interviews.
+**[2-4 days total]**
+
+## How do I apply?
+
+Applications should be sent to [Kirstie Whitaker](mailto:kirstiej@turing.ac.uk) by **Friday 23rd February**. These should
```
Comment (KirstieJane) [inline call-for-champions.md]: Could make a little google form for this? Would be helpful from our end, and also more uniform I think...

File: call-for-champions.md (hunk 2 above)
Comment (martintoreilly) [reply call-for-champions.md]: We're now 50% RSEs and 50% Data Scientists so needed something to cover both. Happy to change to "support from Turing Research Software Engineers" if that appeals more / avoids a speedbump when parsing.

File: call-for-champions.md (hunk 4 above)
Comment (martintoreilly) [reply call-for-champions.md]: Done in commit #417fa57

File: call-for-champions.md (hunk 5 above)
Comment (martintoreilly) [reply call-for-champions.md]: Agreed.

**Comment by martintoreilly** [comment]
I'll create a Google form for the application this evening. I'm thinking about providing the option to either provide a link or upload the paper / data / code. Any concerns about people uploading data they shouldn't to Google drive?

**Comment by martintoreilly** [comment]
I've had a first pass at making a [Google form](https://goo.gl/forms/9znG3PFyBjuNNhM22). You should have received an editor invite to your Turing email. Comments / tweaks welcome.

File: call-for-champions.md (hunk 2 above)
Comment (KirstieJane) [reply call-for-champions.md]: :tada: - no need to change - thanks for explaining :)

**Comment by KirstieJane** [comment]
Hey @martintoreilly - I've made a few little changes - looks great to me & ready to rock 🤘. THANK YOU!

Re: your questions above:

> Should the deadline be midnight Sunday?

Yes, let's move the deadline to Sunday and then tell people if they ask that *really* the deadline is "before Kirstie & Martin arrive at work on Monday morning".

>Can you review my first pass estimates of the time commitment required from champions and the amount of RSE support we're promising (we've about 16 weeks total in our remaining budget)?

I think it looks great.

> Reproducible Research Champions is quite long winded and I'm not sure about using Champions alone (which I've done a lot). What do you think about using Reproducibility Champions everywhere?

I don't know quite why I don't love "Reproducibility" as much, but I totally hear you on "Reproducible Research Champions" being really long! I think it's totally fine to use "Champions" as a short hand, and I have no problem with shortening to "Reproducibility Champions" either.

Short answer: all of the above are totally fine 🤷‍♀️

File: call-for-champions.md (hunk 3 above)
Comment (martintoreilly) [reply call-for-champions.md]: Added section in commit 36ccd3f.

**Comment by KirstieJane** [comment]
@martintoreilly - I'm now re-thinking my move of putting this file in a comms folder! I think because we'll be linking to this maybe it should go in a `champions` folder instead?

I'm not going to edit at the same time as you, and I have to jump into a meeting in 9 mins, so just make your own call for where it should live and I'll be totally happy with that 😸

**Pull request merged**